onnxruntime/test/python/quantization/test_op_matmul.py

```diff
@@ -347,7 +347,7 @@ def quantize_matmul_u8u8(self, tt, opset, ir_version):
     def test_quantize_matmul_u8u8(self):
         self.quantize_matmul_u8u8(onnx.TensorProto.FLOAT, 18, 8)
 
-    @unittest.skip(reason="Shape inference bug, see onnx PR #5709")
+    @unittest.skip(reason="QLinearMatMul(21), which supports float16, is not implemented in ORT.")
```

Comment:
Local testing shows this did not fail due to a shape inference bug (on onnx 1.16), so updated documentation here.